DEVELOPING.md

```diff
@@ -17,7 +32,131 @@ This project:
 
 3. uses [Swift Package Manager](https://github.com/apple/swift-package-manager) to manage dependencies for development. Cocapods is supported for *consuming* the SDK.
 
-4. uses [secret-shield](https://github.com/mapbox/secret-shield) to help block secrets such as access tokens from being exposed. Install `secret-shield` by entering `npm install -g @mapbox/secret-shield`. Install the pre-commit hook by running `scripts/install-pre-commit/install-pre-commit.sh`
+4. uses [secret-shield](https://github.com/mapbox/secret-shield) to help block secrets such as access tokens from being exposed. Setup `secret-shield` by running:
+```
+npm install -g @mapbox/secret-shield
+scripts/install-pre-commit/install-pre-commit.sh
+```
+
+5. uses CircleCI and AWS Device Farm for continuous integration.
+
+### Accessing the Maps SDK's source
+
+Clone the git repository:
+
+```
+git clone git@github.com:mapbox/mapbox-maps-ios.git && cd mapbox-maps-ios
+```
+
+### Building the Maps SDK
+
+Pre-requisite: Valid `.netrc` file located on your machine at `~/.netrc`. Please visit https://docs.mapbox.com/ios/maps/overview for additional information about setting up your `.netrc` file.
+
+Open `Package.swift` and build the MapboxMaps target.
+
+### Update dependency versions
+
+You must remember to update the Package.swift file in https://github.com/mapbox/mapbox-maps-ios/blob/main/Package.swift, not the App.xcworkspace, using the following commands:
+
+```
+xed Package.swift
+//update versions in there
+swift package update
+```
```

Comment:
Updating dependencies has more steps:

* Update the dependency versions in Package.swift
* Open Package.swift in Xcode and resolve dependencies. This updates Package.resolved. Close that Xcode workspace.
* Open Apps/Apps.xcworkspace in Xcode and resolve dependencies. This updates a different Package.resolved file in the Apps.workspace bundle. Close that Xcode workspace.
* Update the dependency versions in MapboxMaps.podspec
* Update the dependency versions in `scripts/release/packager/versions.json`

Commit the changes to all 5 files and open a PR.